### PR TITLE
Add gisele name and url

### PR DESCRIPTION
**Pull request by formosafan**
Miles大神最後14堂星期四的課

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
 # Git Workshop
 
 This workshop is for practice for Git [presentation](https://docs.google.com/presentation/d/1MmazSNHtaWM6eL-R0OvY6AYRkKqKUrDHn53a52wxDbE).
+
+
+
+[Gisele](https://github.com/formosafan/git-workshop)
```
Comment (iyatween) [inline README.md]: !!!!!!!

File: README.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
 # Git Workshop
 
 This workshop is for practice for Git [presentation](https://docs.google.com/presentation/d/1MmazSNHtaWM6eL-R0OvY6AYRkKqKUrDHn53a52wxDbE).
+
```
Comment (iyatween) [inline README.md]: 安鳥微啊

Comment (formosafan) [inline README.md]: whats wrong

Comment (MilesChou) [inline README.md]: @akayhu Hi

**Pull request merged**